ui/src/app/applications/components/application-operation-state/application-operation-state.tsx

```diff
@@ -16,7 +16,7 @@ interface Props {
     operationState: models.OperationState;
 }
 const buildResourceUniqueId = (res: Omit<models.ResourceRef, 'uid'>) => `${res.group}-${res.kind}-${res.version}-${res.namespace}-${res.name}`;
-const FilterableMessageStatuses = ['configured', 'unchanged'];
+const FilterableMessageStatuses = ['changed', 'unchanged'];
```

Comment:
If these do not represent the actual text we are searching for, then perhaps we should capitalize the first letter:
`['Changed', 'Unchanged']`

Also, this will make it consistent with the other three existing filters.